config.toml

```diff
@@ -0,0 +1,19 @@
+username = "my-user"
```

Comment:
Also, let's rename this file to the `example-config.toml`, just to show, that it's an example, not a real configuration.